### ISSUE TITLE
Add language: Hindi

### ISSUE THREAD

**Issue by mzraly** [jzohrab/lute-v3#75]
**NOTE: this form is for you to share your language settings with other new users.  If you're a GitHub user, it would be super if you could instead create a Pull Request with your language settings, using the files in this repo's `lute/db/demo/language` and `/stories` as references.  Thanks!**

If your language settings are working well for you, please share them with other new users by providing the following data:

* name: Hindi1
* dict_1: https://www.boltidictionary.com/en/search?s=###
* dict_2: https://www.bing.com/translator/?from=hi&to=en&text=###
* sentence_translation: https://www.bing.com/translator/?from=hi&to=en&text=###
* show_romanization: true
* right_to_left: false
* parser_type: spacedel
* split_sentences: .!?|
* split_sentence_exceptions: Mr.|Mrs.|Dr.|[A-Z].|Vd.|Vds.
* word_chars:a-zA-Z\u0900-\u097F
* character_substitutions: ´='|`='|’='|‘='|...=…|..=‥

It's good to have a short demonstration story available as well.  Please paste a family-friendly (!) story below:

```
Title: Wikipedia Hindi article sample text (from UN Universal Declaration of Human Rights)
Source: https://en.wikipedia.org/wiki/Hindi

अनुच्छेद १(एक): सभी मनुष्य जन्म से स्वतन्त्र तथा मर्यादा और अधिकारों में समान होते हैं। वे तर्क और विवेक से सम्पन्न हैं तथा उन्हें भ्रातृत्व की भावना से परस्पर के प्रति कार्य करना चाहिए।
```

**Comment by mzraly** [comment]
Actually this has some issues.  The word_chars field should skip \u0964 and \u0965 (DANDA and DOUBLE DANDA), which should be added to split_sentences.

Second, it doesn't seem like the split_sentences field accepts Unicode escape sequences, so they have to be copied directly.

Third, the dict_2 bing URL doesn't see to work, for some reason the application includes the application URL and makes the substitution in the wrong place, e.g. http://localhost:5000/bing/search/12/और/https://www.bing.com/translator/%3Ffrom=hi&to=en&text=%23%23%23, which looks like a combination of image lookup and dict_2 lookup with the ### (%23%23%23) not modified in the dict_2 url.

**Comment by jzohrab** [comment]
Addressed by PR #76, thank you!
